src/services/bundle_downloader.js

```diff
@@ -7,7 +7,7 @@ This Source Code Form is subject to the terms of the Mozilla Public License, v.
 This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
 */
 
-export default class EntityDownloader {
+export default class BundleDownloader {
```

Comment:
I'd consider fetching bundle metadata from `/bundle/{bundleId}/info` endpoint. It's no longer included in bundle itself, but some data may be useful, like `storagePeriods`

Reply:
Btw `storagePeriods` is an argument in `storeBundle` method. For now all Atlases have bundle and bundle metadata as follows:
```
{
    bundleId: <bundleId>
    storagePeriods: null
}
```
We should fetch `storagePeriods` or change `storeBundle` method